dapos/dapos_transport_http.go

```diff
@@ -114,17 +106,27 @@ func (this *DAPoSService) getTransactionsByToAddressHandler(responseWriter http.
 	responseWriter.Write([]byte(response.String()))
 }
 
-// getTransactionsByFromAddressHandler
+// getTransactionsHandler
 func (this *DAPoSService) getTransactionsHandler(responseWriter http.ResponseWriter, request *http.Request) {
 	response := this.GetTransactions()
 	setHeaders(&responseWriter)
 	responseWriter.Write([]byte(response.String()))
 }
 
+// getReceiptHandler
+func (this *DAPoSService) getReceiptHandler(responseWriter http.ResponseWriter, request *http.Request) {
+	vars := mux.Vars(request)
+	response := this.GetReceipt(vars["hash"])
+	setHeaders(&responseWriter)
+	responseWriter.Write([]byte(response.String()))
```

Comment:
With this release, this endpoint is deprecated (but should continue to exist here, as it does).
Should we notify the consumer of that in some way? Is there a convention we could follow?